static/ca_bydate.js

```diff
@@ -250,8 +281,7 @@ function parsingDone(results, file) {
     height:"600px",
     layout:"fitColumns",
     initialSort:[
-      {column:"date", dir:"asc"},
-      {column:"county", dir:"asc"},
+      {column:"date", dir:"desc"}
```

Comment:
out of curiosity why did you take out the `county` sort?  intentional?

Reply:
I saw that the request was to sort by date first and then by visit_index. I took out county and and added visit_index. For some reason, when we do that it only sorts by visit_index and not by date). Tried it in different ways and if we had date and visit_index, it would always ignore date. Had to push the code before I could find a workaround, so I kept the initial sort by date.